public/app/features/alerting/unified/components/rule-editor/LabelsField.tsx

```diff
@@ -18,6 +18,9 @@ interface Props {
   dataSourceName?: string | null;
 }
 
+// Some labels are used internally and should not be displayed to the user
+const restrictedLabelKeys = new Set(['__grafana_origin']);
```

Comment:
why do we need a set here if we only use one element?

Reply:
and, I would move these restricted labels set to a constant outside the component

Reply:
I guess we might want to add more in the future like `rule_uid`, `alertname` etc
Its outside of the component